source/manual/github-unavailable.html.md

```diff
@@ -6,59 +6,67 @@ layout: manual_layout
+1. Follow the [CodeCommit guide](/manual/howto-checkout-and-commit-to-codecommit.html#quick-reference-guide) to clone the repo and commit/push changes if needed.
+1. Set the image tag, image registry and image repository names that you will use by running the following commands. You don't need to modify any of the values in the commands.
 
 ```
-LOCAL_HEAD_SHA="$(git rev-parse HEAD)"
+LOCAL_HEAD_SHA=$(git rev-parse HEAD)
 IMAGE_TAG="release-${LOCAL_HEAD_SHA}"
-ECR_REGISTRY="172025368201.dkr.ecr.eu-west-1.amazonaws.com"
-REPO=$(awk -F '=' /GOVUK_APP_NAME/'{print $2}' Dockerfile)
+REGISTRY="172025368201.dkr.ecr.eu-west-1.amazonaws.com"
+REPO=$(basename "$PWD")
 ```
 
-1. Build the docker image locally with the tag named after the latest sha.
+1. Build the container image and tag it appropriately.
 
 ```
- docker build --platform linux/amd64 -f Dockerfile -t $IMAGE_TAG .
+docker build --platform linux/amd64 -t $IMAGE_TAG .
 ```
 
-1. Log in and push the image to ECR:
+1. Log into ECR and push the image:
 
 ```
-gds aws govuk-production-poweruser aws ecr get-login-password --region eu-west-1 | docker login --username AWS --password-stdin $ECR_REGISTRY
-docker push $ECR_REGISTRY/$REPO:$IMAGE_TAG
+gds aws govuk-production-poweruser aws ecr get-login-password --region eu-west-1 | docker login --username AWS --password-stdin $REGISTRY
+docker push $REGISTRY/$REPO:$IMAGE_TAG
 ```
 
-### Deploy the image to EKS using Argo
-
-1. Get the Admin Argo credentials by running `kubectl -n cluster-services get secret argocd-initial-admin-secret -oyaml |yq .data.password |base64 -d`
-1. Log into [Argo](https://argo.eks.integration.govuk.digital/)
-1. Search for the application you wish to deploy in the "Search applications..." search bar
-1. Select the application from the list below the search bar
-1. Select "APP DETAILS"
-1. Go to "DISABLE AUTO-SYNC" then select "ok" when prompted "Disable Auto-Sync?"
-1. Close the "APP DETAILS" menu, then select the three dots next to the Deploy object for the part of the application you'd like to redeploy. For example, `account-api-worker` would be for deploying the `account-api-workers`.
-1. Go to the "LIVE MANIFEST" and select "EDIT". Find the "image", then update it to the new image tag you'd like to deploy. (You can double-check you have the correct image name by doing a `git log` after [checking out the code from CodeCommit](/manual/howto-checkout-and-commit-to-codecommit.html)).
-1. Click "Save" and Argo will start the deployment, which should complete in under ten minutes.
+### Deploy the image to Kubernetes using Argo CD
+
+> ⚠️ This procedure will disable all automatic deployments not only in the
+> environment you are working on but also in higher environments. For example,
+> performing this procedure on integration will prevent all automatic
+> deployments in integration, staging and production.
```

Comment:
Nitpick: this can all be on one line (i.e. no hard line-breaks), for consistency with the rest of the document.

Reply:
ok, the rest of the doc already isn't consistent though.

Reply:
Actually on second thoughts, nobody should have to spend time on this. I'll reformat the repo and add an mdlint check so that nobody else has to waste time on this in future 👍